changelog/v1.18.0-beta34/validate-large-configs.yaml

```
@@ -0,0 +1,10 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/solo-projects/issues/7089
+    resolvesIssue: false
+    description: >-
+      Fix the validation of large configurations. 
+      Previously, the configuration was being passed as an argument and we were 
+      seeing `argument list too long`. 
+      This change passes the configuration as a file instead, using STDIN (/dev/fd/0)
+      avoiding the need to store the file on disk.
```

This is written very much for engineers in the format I would think of for the pr description itself. 
However this is customer facing.

```suggestion
      Fix the validation of large configurations when using envoy validation.
      Previously if the configuration grew too large translation would be blocked unless Transformation Validation was disabled.
      This change passes the configuration as a file instead, using STDIN (/dev/fd/0)
      avoiding the need to store the file on disk.
```